specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +555,183 @@
           }
         }
       }
+    },
+    "/weather/historical/actuals/daily/{format}": {
+      "get": {
+        "description": "**Get Daily Historical Actuals**\n  \n  \n**Applies to**: G1 and G2 pricing tiers.\n\n\nThe service returns climatology data such as past daily actuals for a given coordinate location.",
+        "operationId": "Weather_GetDailyHistoricalActuals",
+        "x-ms-examples": {
+          "DailyHistoricalActuals": {
+            "$ref": "./examples/GetDailyHistoricalActuals.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "name": "query",
+            "description": "Coordinates of the location for which current conditions information is requested. The applicable query is specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
```

Comment:
consider a refined non-string version as in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Weather/preview/1.0/weather.json#L99-L114